tool/python/examples/user1-cifar10/main.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
```

Comment:
Can we put these two files in a separate branch, since they are used only by Rafiki currently?